### ISSUE TITLE
Tools: add support for PHPUnit Polyfills

### ISSUE THREAD

**Issue by jrfnl** [shivammathur/setup-php#502]
**Describe the feature**
Would be great if the PHPUnit Polyfills Composer package would be available straight from tools.

Ref: https://github.com/Yoast/PHPUnit-Polyfills

**Note**: PHPUnit Polyfills has a (Composer) dependency on PHPUnit, but once installed, tests can be run both via Composer as well as via a PHPUnit PHAR file.
I'm not sure whether that would add extra complexity to an implementation for this ?

**Full disclosure**: I'm the author of the PHPUnit Polyfills package.

The package was released end of October last year and has reached (nearly) one million downloads already, so the uptake of projects using it is extensive (and still growing).

**Version**
- [x] I have checked releases, and the feature is missing in the latest patch version of `v2`.

**Underlying issue**
This tool is typically used to make test suites cross-version compatible with multiple PHPUnit versions.

**Are you willing to submit a PR?**
Yes, happy to, if you are open to adding the tool.

**Comment by shivammathur** [comment]
@jrfnl 

Go ahead with the PR 🚀 

This should work.

```json
"phpunit-polyfills": {
  "type": "composer",
  "repository": "yoast/phpunit-polyfills"
}
```

**Comment by jrfnl** [comment]
Closed as fixed via #503
